scripts/Synapse/ReRunECRfromPostBundle.ipynb

```diff
@@ -0,0 +1,82 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "962a47d0-b2c4-4516-84c0-b947280645fe",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datetime\n",
+    "from pyspark.sql import SparkSession\n",
+    "from pyspark.sql.functions import lit, current_timestamp\n",
+    "from notebookutils import mssparkutils\n",
+    "\n",
+    "spark = SparkSession.builder.getOrCreate()\n",
+    "\n",
+    "# source and destination paths\n",
+    "storage_account = \"$STORAGE_ACCOUNT\"\n",
+    "ecr_post_bundle_file_path = f\"abfss://bundle-snapshots@{storage_account}.dfs.core.windows.net/post/ecr\"\n",
+    "ecr_rerun_file_path = f\"abfss://source-data@{storage_account}.dfs.core.windows.net/ecr-rerun\"\n",
+    "\n",
+    "# parquet log file: timestamp, filename, and destination path\n",
+    "timestamp_str = datetime.datetime.now().strftime(\"%Y%m%d_%H%M%S\")\n",
+    "parquet_file_name = f\"copied_files_log_{timestamp_str}.parquet\"\n",
+    "parquet_file_path = f\"{ecr_rerun_file_path}/logs/{parquet_file_name}\"\n",
+    "\n",
```

Comment:
We don't want to store the logs in `source-data/ecr-rerun` since we have a trigger that will attempt to run this log file through record linkage and then the message parser. I believe you could save it one level up to `source-data/ecr-rern-logs` (or something like that) and it won't trigger anything. But we should test this in dev.

Reply:
that shouldn't be a problem to fix; i'll move it a level up i think that's a good idea 💡